setup_sub.py

```diff
@@ -23,7 +23,10 @@
 from distutils.core import setup
 
 
-VERSION = '0.7.1'
+from pyne.pyne_version import PYNE_VERSION
+VERSION = PYNE_VERSION
```

Comment:
Looks like we can save a line and use `PYNE_VERSION` in the `setup_kwargs` dictionary below.